src/renderer/src/components/Messages/VoiceInput.tsx

```
@@ -69,73 +67,126 @@ export default function VoiceInput({
   const status = useSelector((state: RootState) => state.audios.status);
   const audios = useSelector(selectAudios);
 
-  const sendMessage = () => {
-    let text = '';
-    audios.forEach((audio) => {
-      if (audio.transcription) {
-        text += audio.transcription + ' ';
+  const sendMessage = async () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)